src/Twig/SecurityPolicyContainerAware.php

```diff
@@ -71,7 +71,7 @@ public function __construct(ContainerInterface $container, array $extensions = [
         $this->extensions = $extensions;
     }
 
-    public function checkSecurity(array $tags, array $filters, array $functions): void
+    public function checkSecurity($tags, $filters, $functions): void
```

Comment:
The return type declarations "bool", "void", are not present in the interface:

https://github.com/twigphp/Twig/blob/v2.4.8/lib/Twig/Sandbox/SecurityPolicyInterface.php

Reply:
@jmontoyaa yes, and?

Reply:
my bad, it works well :+1: